GiEnJul/Repositories/GenericRepository.cs

```diff
@@ -50,7 +50,7 @@ protected async Task<T> DeleteAsync(T entity)
             }
             catch (Exception e)
             {
-                _log.Error("Exception while trying to delete Entity:{@entity}, in table:{@tablename}. \n{@Exception}", entity, _table.Name, e);
+                _log.Error(e, "Exception while trying to delete Entity:{@entity}, in table:{@tablename}", entity, _table.Name);
```

Comment:
To avoid getting the exception object mixed into the params[] for string formatting, we can pass it as the first argument in log.Error or log.Fatal.